app/src/main/java/org/wikipedia/main/MainFragment.kt

```diff
@@ -547,6 +552,21 @@ class MainFragment : Fragment(), BackPressedHandler, MenuProvider, FeedFragment.
         }
     }
 
+    private fun maybeShowImportReadingListsNewInstallDialog() {
+        if (!Prefs.importReadingListsNewInstallDialogShown) {
+            ReadingListsFunnel().logReceiveStart()
+            AlertDialog.Builder(requireContext())
+                .setTitle(R.string.shareable_reading_lists_new_install_dialog_title)
+                .setMessage(R.string.shareable_reading_lists_new_install_dialog_content)
+                .setPositiveButton(R.string.shareable_reading_lists_new_install_dialog_learn_more) { _, _ ->
+                    // TODO: open external browser
```

Comment:
Are we still waiting on this FAQ url?

Reply:
good catch - the latest design no longer has a "learn more" button.